Open Judge System/Web/OJS.Web/Areas/Administration/ViewModels/Contest/ChangeParticipationEndTimeViewModel.cs

```diff
@@ -0,0 +1,86 @@
+﻿namespace OJS.Web.Areas.Administration.ViewModels.Contest
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using System.Data.Entity.SqlServer;
+    using System.Linq.Expressions;
+    using System.Web.Mvc;
+
+    using OJS.Data.Models;
+
+    using Resource = Resources.Areas.Administration.Contests.Views.ChangeTime;
+
+    public class ChangeParticipationEndTimeViewModel
+    {
+        private static readonly int DefaultBufferTimeInMinutes = 30;
+
+        public ChangeParticipationEndTimeViewModel()
```

Comment:
IMO, when empty default constructor is neccessary,best to comment it with something like " // Default constructor needed for REASON". Otherwise it is just wierd.

Reply:
I think in this case it is not needed, since it's usage location can be easily tracked in the same file and if removed, a build error emerges immediately.